lib/stream.js

```diff
@@ -3,14 +3,18 @@
 var Writable = require('stream').Writable;
 var util = require('util');
 
-function StreamWrapper(parser) {
+function StreamWrapper(parser, isLexer) {
     Writable.call(this);
     this._parser = parser;
+    this._isLexer = isLexer;
 }
 
 util.inherits(StreamWrapper, Writable);
 
 StreamWrapper.prototype._write = function write(chunk, encoding, callback) {
+    if (this._isLexer) {
+        chunk = '@{% ' + chunk.toString() + ' %} @lexer lexer';
```

Comment:
This is ugly, but I wanted to be able to write a valid javascript file containing the lexer, and this was the lowest friction change to get that. So now the lexer file can look like:
```
const moo = require("moo");

const lexer = moo.compile({
    ws:     /[ \t]+/,
    number: /[0-9]+/,
    word: /[a-z]+/,
    times:  /\*|x/
});
```
instead of 
```
@{%
const moo = require("moo");

const lexer = moo.compile({
  ws:     /[ \t]+/,
  number: /[0-9]+/,
  word: /[a-z]+/,
  times:  /\*|x/
});
%}

@lexer lexer
```
Since we're just doing string concatenation here it's not very robust in that it doesn't check that the `lexer` object is actually named `lexer`. The easiest solution is to have a lint check in jstestfuzz that `lexer` is called such.

Reply:
🤔 maybe have the file `export default mylexername` and then rely on js modules?

Reply:
What do you mean by `rely on js modules`?

Reply:
> One thing I do think we should do for our own lexer.js files is set module.exports = {lexer} so we can have automated testing of the lexers independently.

👍